service/raft/raft_group_registry.hh

```diff
@@ -49,6 +51,50 @@ struct raft_server_for_group {
     raft_rpc& rpc;
     raft_sys_table_storage& persistence;
     std::optional<seastar::future<>> aborted;
+    std::optional<lowres_clock::duration> default_op_timeout;
+};
+
+struct raft_timeout {
+    seastar::compat::source_location loc = seastar::compat::source_location::current();
+    std::optional<lowres_clock::duration> value;
```

Comment:
Better to use time_point here. No need to recalculate duration in case of retrying an operation.